Draft/Classes/HTTP.swift

```diff
@@ -12,4 +12,4 @@ public enum HTTPMethod: String {
 }
 
 public typealias HTTPHeaders = [String : String]
-public typealias HTTPParameters = [String : Any]
+public typealias HTTPParameters = [String : CustomStringConvertible]
```

Comment:
This is okay only if typical values we pass in as parameters are custom string convertible already

Reply:
Yes I believe all of the Foundation classes are